mthree/mitigation.py

```diff
@@ -858,6 +858,9 @@ def _job_thread(jobs, mit, qubits, num_cal_qubits, cal_strings):
             counts.extend(_counts)
     # attach timestamp
     timestamp = res.date
+    # Timestamp can be None
+    if timestamp is None:
+        timestamp = datetime.datetime.now()
     # Needed since Aer result date is str but IBMQ job is datetime
     if isinstance(timestamp, datetime.datetime):
         timestamp = timestamp.isoformat()
```

Comment:
In the event that `res.date == None`, the `datetime.datetime` timestamp will be converted to a `str` in this line here, so it would not fail. Should it be more explicit by doing the string conversion directly (i.e., `timestamp = datetime.datetime.now().isoformat()`)?

Reply:
Yeah, no my previous comment was wrong.  I skipped line 864 in my brain thinking it was for Aer only, but it does indeed capture the new code as well.